diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 0000000..1111111 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -47,6 +47,7 @@
             ReactAndroid::jsi
             fbjni::fbjni
             android
+            c++_shared
         )
     else()
         target_link_libraries(
@@ -69,6 +70,7 @@
                 ReactAndroid::react_render_observers_events
                 fbjni::fbjni
                 android
+                c++_shared
         )
     endif()
 else()
@@ -75,5 +77,6 @@
     target_link_libraries(rnscreens
         ReactAndroid::jsi
         android
+        c++_shared
     )
 endif()
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index 0000000..1111111 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -45,5 +45,6 @@
     ReactAndroid::reactnative
     ReactAndroid::jsi
     fbjni::fbjni
+    c++_shared
   )
 else()
